deploy/efs-csi/04-clusterrolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+# Lets dedicated-admins create a PersistentVolume
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: system:persistent-volume-provisioner
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:persistent-volume-provisioner
```

Comment:
What does this role grant that dedicated admins don't already have?

Reply:
Dedicated admins cannot create PVs today, this setup requires the customer to create PVs.  Else SRE has to do it for them.. middle man kind of role.

Reply:
Right.